scripts/sexbound/plugins/pregnant/pregnant.lua

```diff
@@ -448,10 +446,14 @@ end
 --- Returns whether or not the other actor is a compatible species
 function Sexbound.Actor.Pregnant:otherActorIsCompatibleSpecies(otherActor)
     local actor = self:getParent()
-    self:getLog():debug("Species check: "..actor:getSpecies().." on "..otherActor:getSpecies())
+    local ownSpecies = actor:getOffspringSpecies()
```

Comment:
That the offspringSpecies needs to be considered for baby generation, so that we don't end up trying to generate a non-spawnable object baby makes sense.

However, determining the general compatibility between two species should be left at the actual species. Otherwise people have to dig even deeper to find the species name they need to add to the compatibleSpecies list in order to enable pregnancies between two species for themselves, when they now also have to consider that someone somewhere defined a secondary species name for offspring that is used.